packages/aurora-feed/index.js

```diff
@@ -85,7 +93,9 @@ class Feed extends React.Component {
     this.setState({
       inputEditorState: editorState
     });
-    this.searchCard(editorState);
+    // Only search once every XYZ miliseconds so we're not flashing
+    const searchOnlyAfterABit = _.debounce(this.searchCard, 250);
```

Comment:
what does an _ do?

Reply:
`_` is [lodash](https://lodash.com/), which is the most commonly used utility library in javascript. Think of it like Apache Commons in Java. It's got a bunch of useful tools for working with arrays, functions, and objects. 

In this case it has a debounce function!